support-frontend/migration/README.md

```diff
@@ -22,11 +22,7 @@ All files should replace any existing versions of the same file in their destina
 
 ### Post-migration manual fixes
 
-There are a few manual syntax fixes that need to be applied. Search and replace in both /assets and /stories for the following:
-
-- Replace “import type” with “import
-
-Then do the following file-specific changes:
+There are a few manual syntax fixes that need to be applied:
```

Comment:
Interestingly after updating the `.eslintrc.js` to use `@guardian/eslint-config-typescript` every time I attempted to manually replace  “import type” with “import” my IDE would apply our lint rules on saving the file and this began reintroducing the `type` keyword in import statements! 

I believe the `@typescript-eslint/consistent-type-imports` rule from `@guardian/eslint-config-typescript`'s [configuration](https://github.com/guardian/configs/blob/main/packages/eslint-config-typescript/index.js#L40) is driving this. The use of the `type` keyword in these import statements is correct syntax in Typescript.

I therefore attempted to test and build (ssr, dev and prod) the source code with the `type` keyword in place and all the commands worked. I also ran the site locally and it all behaved as expected. I therefore think we can remove this step.